add integration tests target

mlevesquedion opened the pull request:
Fixes #316.

I'm assuming that `CONTRIBUTING.md` represents the desired state, that is to say an `integration-tests` target is desired. If that's not the case, I'm happy to edit `CONTRIBUTING.md` to indicate how to actually run the integration tests.

danielpacak reviewed (changes requested):
:wave: @mlevesquedion Thanks for spotting this out and opening PR! I think I'd rather update CONTRIBUTING.md and explain that we have separate test suits for Starboard CLI and Starboard Operator.

We run them separately in our CI workflow after all:
- https://github.com/aquasecurity/starboard/blob/main/.github/workflows/build.yaml#L68
- https://github.com/aquasecurity/starboard/blob/main/.github/workflows/build.yaml#L97

Beyond that, the first suite may leave the cluster in dirty state, and we may end up with flaky integration tests. WDYT?

mlevesquedion commented:
Hello @danielpacak!

That makes sense to me. Thanks for pointing out the workflows.

I've removed the `make` target and updated `CONTRIBUTING.md` instead. PTAL.

danielpacak reviewed (approved):
Looks great! Thank you @mlevesquedion !
